static/js/tools/scatter/info.tsx

```diff
@@ -130,7 +146,8 @@ function Info(): JSX.Element {
           </a>
         </li>
         <li>
-          <b>Bachelor Degree Attained vs Females Per Capita</b> for counties in{" "}
+          <b>Bachelor&apos;s Degree Attainment vs. Female Population Per Capita</b>{" "}
```

Comment:
nit: s/Female/Women/ ?

Reply:
Population of Women Per Capita?
We do use the term "Female" in many other places ...

Reply:
+1

Reply:
i did not update this yet.. lmk what you think

Reply:
For consistency, we can leave it as female for now.

Reply:
updated, ptal

Reply:
lg, thx